package/MDAnalysis/topology/PDBParser.py

```diff
@@ -515,3 +515,94 @@ def _parse_conect(conect):
     bond_atoms = (int(conect[11 + i * 5: 16 + i * 5]) for i in
                   range(n_bond_atoms))
     return atom_id, bond_atoms
+
+def fetch_pdb(PDB_IDS=None,
```

Comment:
I'd propose to shorten the name to `fetch()`.

It also suggests how to do the same functionality for other formats: `FORMAT.fetch()`.